python/ccp.py

```diff
@@ -3,6 +3,33 @@
 import time
 import struct
 from enum import IntEnum, Enum
+from typing import Optional
+from typing_extensions import TypedDict
+
+from . import Panda
+
+class StationIdsResponse(TypedDict):
+  id_length: int
+  data_type: int
+  available: int
+  protected: int
+
+class DAQListSizeResponse(TypedDict):
+  list_size: int
+  first_pid: int
+
+class SessionStatusResponse(TypedDict):
+  status: int
+  info: Optional[int]
+
+class DiagnosticServiceResponse(TypedDict):
+  length: int
+  type: int
+
+class ActionServiceResponse(TypedDict):
+  length: int
+  type: int
```

Comment:
If anyone has better names for these - i'm not too familiar with the protocol yet 😅